### PR TITLE
Updated solution after Ana's review.

### PR DESCRIPTION
**Pull request by dianatanase**
Changed return type of alphabet to LinkedList.
Made createVertices return void, added a get method instead for testing.
Updated tests according to these changes.

### CODE REVIEW

File: group/assignment5/UnknownAlphabet.java [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+	
+	// Assume dictionary input is a List.
+	public LinkedHashSet<Character> getAlphabet(List<String> dict) throws Exception {
+
+		LinkedHashSet<Character> ordered_alphabet = new LinkedHashSet<Character>();
+
+		if (dict.isEmpty())
+			return ordered_alphabet;
+
+		// Return all characters (no duplicates) if dictionary has only one word.
```
Comment (comRamona) [inline group/assignment5/UnknownAlphabet.java]: Could we move this logic to createVertices() maybe? I don't insist though

**Review by comRamona — APPROVED** [review]
I'm happy with this

File: group/assignment5/UnknownAlphabet_test.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import static org.junit.Assert.*;
+
+import java.util.LinkedList;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+
+public class UnknownAlphabet_test {
```
Comment (comRamona) [inline group/assignment5/UnknownAlphabet_test.java]: In Java we usually use camelCase, I would rename this UnknownAlphabetTest

Comment (comRamona) [inline group/assignment5/UnknownAlphabet_test.java]: In Java we usually use camelCase, I would rename this UnknownAlphabetTest

Comment (comRamona) [inline group/assignment5/UnknownAlphabet_test.java]: In Java we usually use camelCase, I would rename this UnknownAlphabetTest

Comment (dianatanase) [reply group/assignment5/UnknownAlphabet_test.java]: Thanks, I'll change it now.

**Pull request closed without merging**